pkgs/development/python-modules/guidata/default.nix

```diff
@@ -62,6 +65,19 @@ buildPythonPackage rec {
     export QT_QPA_PLATFORM=offscreen
   '';
 
+  disabledTests = lib.optionals stdenv.hostPlatform.isDarwin [
```

Comment:
Would have been nice to notify upstream about this. Although they'd probably wouldn't have responded fast enough to solve this, broadly speaking they are very helpful and kind.

Reply:
> Yes you are right. Can you open an issue upstream? Otherwise I can do it if you prefer.

I think you should do it, because I don't have the logs available at hand.